### PR TITLE
Add prod deploy pipeline

### LINKED ISSUE #404: Create a specific environment for Fawkes in production
blocked by #403

### PR DESCRIPTION
**Pull request by JordanGuinn**
Closes #404.

-----------------------------------------------------------------------------------------------------------------------

### Associated PR
- https://github.com/skylight-hq/ctoec-devops/pull/139

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -167,3 +167,22 @@ workflows:
...
+      - hold:
+          type: approval
+      - install:
+          requires: 
+            - hold
+      - build:
+          env: prod
+          requires:
+            - install
+      - prod-check:
```
Comment (jlhogan) [inline .circleci/config.yml:181]: `prod-check` is extra check to ensure we can't accidentally push an un-buildable version to prod?

**Review by jlhogan — APPROVED** [review]
wooooooooooooooo!

**Review by bamader — APPROVED** [review]
LGTM!

File: .circleci/config.yml (hunk 1 above)
Comment (JordanGuinn) [reply .circleci/config.yml:181]: Yes!  And also just a general "hey you're sure you wanna do this" step

**Pull request merged**